Description: Use escapejs filter on JavaScript strings
 The escapejs filter will correctly escape single quotes from JavaScript
 strings which may otherwise trigger syntax errors. In this case, the issue
 was visible using the French translation.
 .
 Also removed a trailing paranthesis coming out of nowhere.
From: Adrien Cunin <adrien.cunin@osones.com>
Date: Tue, 22 Apr 2014 10:54:33 +0000 (+0200)
Subject: Use escapejs filter on JavaScript strings
X-Git-Url: https://review.openstack.org/gitweb?p=openstack%2Fhorizon.git;a=commitdiff_plain;h=232b71286f3b1426412a3ffbed5d7065c2e24275
Origin: upstream, https://review.openstack.org/#/c/92006/
Change-Id: I25001815e17be5afa8eb28cc28c5423e3dcd973c
Bug-Ubuntu: https://launchpad.net/bugs/1311047

diff --git a/openstack_dashboard/dashboards/project/instances/templates/instances/_flavors_and_quotas.html b/openstack_dashboard/dashboards/project/instances/templates/instances/_flavors_and_quotas.html
index 9abfc82..5cb2c55 100644
--- a/openstack_dashboard/dashboards/project/instances/templates/instances/_flavors_and_quotas.html
+++ b/openstack_dashboard/dashboards/project/instances/templates/instances/_flavors_and_quotas.html
@@ -40,8 +40,8 @@
 </div>
 
 <script type="text/javascript" charset="utf-8">
-  some_disabled_msg = '{{_("Some flavors not meeting minimum image requirements have been disabled.")}}';
-  all_disabled_msg = '{{_("No flavors meet minimum criteria for selected image.")}})';
+  some_disabled_msg = '{{_("Some flavors not meeting minimum image requirements have been disabled.")|escapejs}}';
+  all_disabled_msg = '{{_("No flavors meet minimum criteria for selected image.")|escapejs}}';
 
   if(typeof horizon.Quota !== 'undefined') {
     horizon.Quota.initWithFlavors({{ flavors|safe|default:"{}" }});
